style.css

```diff
@@ -15,12 +15,53 @@ body {
   height: fit-content;
 }
 
+.hide {
+  display: none;
+}
+
+#menuBackground {
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  background-color: #6070ffde;
+  background-size: cover;
+  top: 0%;
+  margin-top: 0;
+  z-index: 50;
+}
```

Comment:
- [ ] As the previous code reviewer aske  I think here you need to blur the background to match the Figma design as much as possible ✅✅